deploy/kubernetes/base/daemonset.yaml

```diff
@@ -0,0 +1,47 @@
+apiVersion: apps/v1beta2 # for versions before 1.8.0 use extensions/v1beta1 [apps/v1beta2]
+kind: DaemonSet
+metadata:
+  name: cadvisor
+spec:
+  template:
+    spec:
+      containers:
+      - name: cadvisor
+        image: gcr.io/google-containers/cadvisor:v0.30.2
```

Comment:
Let's use `k8s.gcr.io/cadvisor` instead. It should point to the same place.

Reply:
done